hadoop-ozone/recon/src/main/java/org/apache/hadoop/ozone/recon/tasks/OmTableInsightTask.java

```diff
@@ -295,21 +286,23 @@ private void writeDataToDB(Map<String, Long> dataMap) {
 
     globalStatsDao.insert(insertGlobalStats);
     globalStatsDao.update(updateGlobalStats);
+
+    insertGlobalStats.clear();
+    updateGlobalStats.clear();
   }
 
   /**
    * Initializes and returns a count map with the counts for the tables.
    *
    * @return The count map containing the counts for each table.
    */
-  private HashMap<String, Long> initializeCountMap() {
-    Collection<String> tables = getTaskTables();
```

Comment:
How do we make sure that `tables` is initialised here?

Reply:
Before initialising count map , we are already initialising taskTables in `reprocess`. Please explain what issue you see here.

Reply:
Makes sense, thanks!